acceptance/setup/common/060_Setup_PCP_Client.rb

```diff
@@ -0,0 +1,19 @@
+require 'pxp-agent/test_helper'
+require 'socket'
+test_name 'Set up SSL certs for pcp-client to use'
+
+step 'On master create certs for the host running pcp-client' do
+  hostname = Socket.gethostname
+  on master, puppet("cert generate #{hostname}")
```

Comment:
As this is just a role identity, this could be something like `beaker-controller` and would remove the need to determine the hostname of the beaker host.